app/src/main/java/org/pragyan/dalal18/fragment/CompanyDescriptionFragment.kt

```diff
@@ -26,11 +26,21 @@ class CompanyDescriptionFragment : Fragment() {
                 ?: throw Exception("Invalid activity")
 
         Picasso.get().load(model.getImageUrlFromCompanyName(arguments?.getString(COMPANY_NAME_KEY))).into(companyImageView)
-        companyNameTextView.text = arguments?.getString(COMPANY_NAME_KEY)
+        if (arguments!!.getBoolean(COMPANY_ISBANKRUPT)) {
```

Comment:
Can you do without not-null assertion operator (!!)

Reply:
**it's of non-nullable recicver type of Bundle.**